napari/_qt/layer_controls/qt_layer_controls_base.py

```diff
@@ -6,6 +6,10 @@
 from ...utils.translations import trans
 from ..widgets._slider_compat import QDoubleSlider
 
+MIN_BLENDING_TOOLTIP = trans._(
+    '`minimum` blending mode works best with inverted colormaps with a white background.',
```

Comment:
Will strings in backticks be translated (i.e. here `minimum`)? Is there another way to indicate these are code and the string should be used verbatim/untranslated?